doc/source/ray-air/package-ref.rst

```diff
@@ -126,6 +126,19 @@ Configs
 
 .. autoclass:: ray.air.config.CheckpointConfig
 
+
+.. _air-builtin-callbacks:
+
+Callbacks
+~~~~~~~~~
+
+Keras
+#####
+
+.. autoclass:: ray.air.callbacks.keras.Callback
+    :members:
```

Comment:
Do we want to also include the Tune callbacks here?

Reply:
Yes!